openslides_backend/action/actions/user/json_upload.py

```diff
@@ -153,6 +177,10 @@ def generate_entry(
                         "info": ImportState.GENERATED,
                     }
             self.handle_default_password(entry, state)
+            if entry.get("saml_id", {}).get("value"):
+                entry.pop("password", None)
+                entry.pop("default_password", None)
```

Comment:
A warning would be good that the fields were removed

Reply:
Add a message to the messages, if one field is removed.